packages/devtools_app/lib/src/screens/app_size/app_size_table.dart

```diff
@@ -13,19 +13,28 @@ import '../../ui/colors.dart';
 
 import 'app_size_controller.dart';
 
-const deferredLoadingSupport = false;
+//Temporary feature flag for deferred loading.
+bool get deferredLoadingSupport => true;
 
 class AppSizeAnalysisTable extends StatelessWidget {
   factory AppSizeAnalysisTable({
     required TreemapNode rootNode,
     required AppSizeController controller,
   }) {
-    final treeColumn = _NameColumn(currentRootLevel: rootNode.level);
+    final treeColumn = _NameColumn(
+      currentRootLevel: deferredLoadingSupport && controller.isDeferredApp
```

Comment:
With the changes suggested above, we only need to check `controller.isDeferredApp`